packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -353,21 +354,24 @@ class BottomSheet extends Component {
 
 		const WrapperView = hasNavigation ? View : ScrollView;
 
-		const getHeader = () => (
-			<>
-				<View style={ styles.bottomSheetHeader }>
-					<View style={ styles.flex }>{ leftButton }</View>
-					<Text
-						style={ bottomSheetHeaderTitleStyle }
-						maxFontSizeMultiplier={ 3 }
-					>
-						{ title }
-					</Text>
-					<View style={ styles.flex }>{ rightButton }</View>
-				</View>
-				{ withHeaderSeparator && <View style={ styles.separator } /> }
-			</>
-		);
+		const getHeader = () =>
+			header || (
```

Comment:
This component only allowed a text as the header, this new prop `header` will allow us to add anything we might need to render in the header.